submissions/WEremite/index.html

```diff
@@ -0,0 +1,100 @@
+                <label for="show">
+                    <img src="https://raw.githubusercontent.com/WEremite/html-css-popup/a910c067cacd328d5e11cad9e339dba80e7e71d3/icons/popup-button.png" alt="popup-button" class="popup-button">
+                </label>
+                    <!-- <div class="popup"> -->
+                    <ul class="icons-menu" id="content">
+                        <li class="icons-item">
+                            <a href="#"><img src="https://raw.githubusercontent.com/WEremite/html-css-popup/a910c067cacd328d5e11cad9e339dba80e7e71d3/icons/svg/001-unicorn.svg" alt="unicorn laughing" class="icons-item-img">
+                                <p>Laugh</p>
+                            </a>
+                        </li>
+                        <li class="icons-item">
+                            <a href="#"><img src="https://raw.githubusercontent.com/WEremite/html-css-popup/a910c067cacd328d5e11cad9e339dba80e7e71d3/icons/svg/002-unicorn.svg" alt="unicorn sleeping" class="icons-item-img">
+                                <p>Sleep</p>
+                            </a>
+                        </li>
+                        <li class="icons-item">
+                            <a href="#"><img src="https://raw.githubusercontent.com/WEremite/html-css-popup/516d8a3f5b12b21a175cf7eab325afd9c040b0fa/icons/svg/003-eye%20mask.svg" alt="unicorn mask" class="icons-item-img">
+                                <p>Mask</p>
+                            </a>
+                        </li>
+                        <li class="icons-item">
+                            <a href="#"><img src="https://raw.githubusercontent.com/WEremite/html-css-popup/a910c067cacd328d5e11cad9e339dba80e7e71d3/icons/svg/004-unicorn.svg" alt="unicorn winkles" class="icons-item-img">
+                                <p>Winkle</p>
+                            </a>
+                        </li>
+                        <li class="icons-item">
+                            <a href="#"><img src="https://raw.githubusercontent.com/WEremite/html-css-popup/a910c067cacd328d5e11cad9e339dba80e7e71d3/icons/svg/005-unicorn.svg" alt="unicorn crying" class="icons-item-img">
+                                <p>Cry</p>
+                            </a>
+                        </li>
+                        <li class="icons-item">
+                            <a href="#"><img src="https://raw.githubusercontent.com/WEremite/html-css-popup/a910c067cacd328d5e11cad9e339dba80e7e71d3/icons/svg/006-unicorn.svg" alt="unicorn mermaid" class="icons-item-img">
+                                <p>Mermaid</p>
+                            </a>
+                        </li>
+
+                        <input type="checkbox" id="show2" name="checkbox2" />
+                        <label for="show2" class="second-label">
+                            More
+                        </label>
+                        <ul class="more-icons-menu" id="content2">

```

Comment:
Element `ul` not allowed as child of element `ul`.

Reply:
Fixed